frame/support/procedural/src/construct_runtime/expand/call.rs

```diff
@@ -68,10 +68,10 @@ pub fn expand_outer_dispatch(
 
 		#[derive(
 			Clone, PartialEq, Eq,
-			#scrate::codec::Encode,
-			#scrate::codec::Decode,
-			#scrate::scale_info::TypeInfo,
-			#scrate::RuntimeDebug,
+			#scrate::__private::codec::Encode,
+			#scrate::__private::codec::Decode,
+			#scrate::__private::scale_info::TypeInfo,
+			#scrate::__private::RuntimeDebug,
```

Comment:
Is there a better way? 😄

Reply:
This actually isn't that bad, since macro-generated code is not part of "surface syntax". I think most people do expect code expanded this way to be sort of internal after all -- if you want to convince yourself more, try looking at the `__is_call_part_defined_{num}` macros that gets automatically generated and exported while we process `#[pallet::call]`.